lemur/certificates/models.py

```diff
@@ -439,6 +437,30 @@ def __repr__(self):
         return "Certificate(name={name})".format(name=self.name)
 
 
+class CertificateAssociation(db.Model):
+    __tablename__ = 'certificate_associations'
+    __table_args__ = (
+        Index(
+            "certificate_associations_ix",
+            "domain_id",
+            "certificate_id",
+        ),
+    )
+    domain_id = Column(Integer, ForeignKey("domains.id"), primary_key=True)
+    certificate_id = Column(Integer, ForeignKey("certificates.id"), primary_key=True)
+    ports = Column(Text())
```

Comment:
I know this is just initial version, still one thought.
Postgres support integer array as data type. Are you planning to use that and is there a Column type for that instead of Text().

Reply:
Thanks, you're completely right about this. I realized as a result of this comment that I was never committing the changes to the DB - once I added the commit, it broke. Using an integer array column works perfectly.